api/operator.go

```diff
@@ -43,6 +43,26 @@ type RaftConfiguration struct {
 	Index uint64
 }
 
+// KeyringOpts is used for performing Keyring operations
+type KeyringOpts struct {
```

Comment:
I'd make this an unexported struct `keyringRequest` since it's not part of your interface for the actual API functions.